src/api/test/functional/webui/apidocs_controller_test.rb

```diff
@@ -4,7 +4,7 @@ class Webui::ApidocsControllerTest < Webui::IntegrationTest
   def test_index
     return unless File.exist? '/var/adm/fillup-templates'
     # test relative links
-    visit apidocs_path
+    visit "/apidocs"
```

Comment:
Why don't you use a named route to have the `apidocs_path` method still working?
I mean adding `as: 'apidocs'` in the rewriting of the route. Like this:
```
get 'apidocs(/index)', to: 'webui/apidocs#index', as: 'apidocs'
```